renaissance/scripts/perf/renaissance-run.sh

```diff
@@ -0,0 +1,301 @@
+#!/bin/bash
```

Comment:
Running this failed with the below error:

./renaissance-run.sh --clustertype=minikube -s localhost  -g prakalp23/renaissance1041:latest  -e ./results -w 1 -m 1 -i 1 --iter=1 -r -n default  --cpureq=1.5 --memreq=3152M --cpulim=1.5 --memlim=3152M -b "page-rank" -d 60

(standard_in) 1: syntax error
(standard_in) 1: syntax error
metric_ci = Could not open input file: /home/chandra/benchmarks/renaissance/scripts/../utils/ci.php
/home/chandra/benchmarks/renaissance/scripts/perf/parsemetrics-promql.sh: line 348: [: too many arguments
metric_ci = Could not open input file: /home/chandra/benchmarks/renaissance/scripts/../utils/ci.php